../subprojects/windecor/deco-theme.cpp:11:9: fatal error: 'bits/stdc++.h' file not found
#include<bits/stdc++.h>
        ^~~~~~~~~~~~~~~

--- subprojects/windecor/deco-theme.cpp.orig	2020-12-14 07:51:43 UTC
+++ subprojects/windecor/deco-theme.cpp
@@ -8,7 +8,6 @@
 #include <sys/stat.h>
 #include <unistd.h>
 #include <fstream>
-#include<bits/stdc++.h>
 
 #include "INIReader.h"
 
